### PR TITLE
Style disabled breakpoints in the list

### PR DESCRIPTION
**Pull request by clarkbw**
This creates style classes for breakpoints so I could make the font colour lighter to match the disabled state.  Needs @helenvholmes review.
I also added an option for the paused state so we could style that separately as well but couldn't find a style that I liked.  Maybe @helenvholmes has ideas for this or it can remain the same.  I used CSS animations to make the breakpoint "flash" a background color which worked pretty well but needs more thought into the timing because it distracted from the editor text location.

@jasonLaster I changed the storybook to be 3 breakpoints from 1 so I could have one of each type.  The last, disabled, breakpoint disappears when you check the box and I wasn't sure how to handle this.  I assuming it is the click handler running and then finding that breakpoint doesn't exist.  But I could use some help here.

### CODE REVIEW

File: public/js/components/Breakpoints.js [hunk 1]
```diff
@@ -66,14 +68,18 @@ const Breakpoints = React.createClass({
 
     return dom.div(
       {
-        className: "breakpoint",
+        className: classnames({
+          breakpoint: true,
```
Comment (jasonLaster) [inline public/js/components/Breakpoints.js]: You might not need the true

File: public/js/components/tests/Breakpoints.js [hunk 2]
```diff
@@ -21,14 +21,18 @@ describe("Breakpoint Component", function() {
...
 
     const $el = renderComponent(Breakpoints, "todomvcUpdateOnEnter");
     const breakpoints = getBreakpoints($el);
-    expect(breakpoints.length).to.equal(1);
+    expect(breakpoints.length).to.equal(3);
     expect(getBreakpointLabel(breakpoints[0])).to.equal("113 this.close();");
+    expect(getBreakpointLabel(breakpoints[1])).to
+    .equal("142 return undefined;");
+    expect(getBreakpointLabel(breakpoints[2])).to
+    .equal("169 var name = text.toLowerCase();");
```
Comment (jlongster) [inline public/js/components/tests/Breakpoints.js]: The indentation looks a little weird here. Preferably:

```
expect(getBreakpointLabel(breakpoints[1]))
  .to.equal("142 return undefined;");
```

File: public/js/test/fixtures/todomvc.updateOnEnter.json [hunk 3]
```diff
@@ -112,6 +112,28 @@
...
+      "server2.conn120.source42:169": {
+        "location": {
+          "sourceId": "server2.conn120.source42",
+          "line": 169
+        },
+        "condition": null,
+        "disabled": true,
+        "loading": false,
+        "id": "server2.conn120.50",
+        "text": "var name = text.toLowerCase();"
```
Comment (jasonLaster) [inline public/js/test/fixtures/todomvc.updateOnEnter.json]: Nice. I assume you added these by hand. 

We generate the fixtures with cypress in the tests/integrations/fixtures tests. You can add two more breakpoints in the updateOnEnter test. 

Cypress requires an account to run so once you install it you'll need to fill out an adoptee form and then ping them in their gitter channel so you have early access. . https://gitter.im/cypress-io/cypress

This info should be in the readme :)

**Comment by jlongster** [comment]
We should work with Helen to figure "selected" states in general. The state for a breakpoint that is currently paused is that it is "selected". The problem is that several things in the UI can be "selected": the current source, frame, breakpoint, scope item, etc. We should figure out the right way to highlight everything that makes sense. Some of those may not need to be highlighted at all, like the "selected breakpoint" which doesn't seem very useful to me.

**Comment by jasonLaster** [comment]
@clarkbw looks good. Sorry I can't help more on the click handler. Commenting from my phone.

**Comment by clarkbw** [comment]
Yes, I think the important aspect is where we want to direct the attention
from our users. Especially considering all the various selection and other
settings available. I think the editor is the right place initially. We
should also consider afterward we may want to give a subtle hint to the
buttons for stepping. Somewhere else though, not here.

**Comment by jlongster** [comment]
If you fix the linting error, I'll merge this

**Comment by clarkbw** [comment]
Ok, thanks. I can fix it up this weekend.

**Comment by jasonLaster** [comment]
Created a new issue for discussing the breakpoint UI https://github.com/jlongster/debugger.html/issues/291. I think what you have is good and we should keep it, but worth discussing more in depth as well.

**Comment by clarkbw** [comment]
I'm just waiting on the Cypress account to finish this up.

**Comment by clarkbw** [comment]
Looks good!  @jlongster r?

**Comment by jlongster** [comment]
Looks good! I wish the fixture updates didn't change the IDs every time, but that's a separate problem.

**Comment by clarkbw** [comment]
> I wish the fixture updates didn't change the IDs every time, but that's a separate problem.

Yeah, it makes the diff really hard to comprehend.  Would be nice to have that fixed.

**Pull request merged**